.github/workflows/ci.yml

```
@@ -65,6 +67,76 @@ jobs:
               with:
                   fetch-depth: 1
 
+            - name: Setup
+              id: setup
+              uses: ./.github/actions/setup-nx
+              with:
+                  cache_mode: ${{ github.event.inputs.clean_checkout == 'true' && 'off' || 'rw' }}
+
+            - name: nx build
+              id: build
+              if: steps.setup.outcome == 'success' || steps.setup.outcome == 'skipped'
+              run: |
+                  yarn nx ${{ github.event.inputs.nx_command || 'affected' }} -t build --exclude tag:skip-gha-cache --parallel=3  --exclude all
```

nit: extra space

```suggestion
                  yarn nx ${{ github.event.inputs.nx_command || 'affected' }} -t build --exclude tag:skip-gha-cache --parallel=3 --exclude all
```

Also, not sure if you can combine the `--exclude`s, but not a big deal if not

---

I have another follow-up PR, will make this tweak there - the multiple excludes do work from what I can see